docs/topics/logging.txt

```diff
@@ -398,12 +408,43 @@ configure logging using your own approach), you can set
 :setting:`LOGGING_CONFIG` to ``None``. This will disable the
 configuration process.
 
-.. note::
-    Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the
-    configuration process is disabled, not logging itself. If you
-    disable the configuration process, Django will still make logging
-    calls, falling back to whatever default logging behavior is
-    defined.
+Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the configuration
+process is disabled, not logging itself. If you disable the configuration
+process, Django will still make logging calls, falling back to whatever default
+logging behavior is defined.
+
+Here's an example of how to make the logging system print everything to the
+console without having Django's disabled default loggers swallow entries. It
+may be useful during local development. Set the environment variable
+``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging calls in the console.
+
+.. snippet::
+    :filename: settings.py
+
+    LOGGING_CONFIG = None
+    LOGGING = {
+        'version': 1,
+        'handlers': {
+            'console': {
+                'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+                'class': 'logging.StreamHandler',
+                'formatter': 'debug',
+            },
+        },
+        'formatters': {
```

Comment:
This isn't really necessary if we're aiming for a minimal example; it's a nicety, but the default formatter is reasonable too.